functional/test/test_streams.py

```diff
@@ -362,6 +363,16 @@ def test_to_csv(self):
         result = self.seq.csv(tmp_path).to_list()
         self.assertEqual(expect, result)
 
+    @unittest.skipUnless(system().startswith('Win'), 'Skip CSV test if not on Windows')
+    def test_to_csv_win(self):
+        tmp_path = 'functional/test/data/tmp/output.txt'
+        elements = [[1, 2, 3], [4, 5, 6], ['a', 'b', 'c']]
+        expect = [['1', '2', '3'], [], ['4', '5', '6'], [], ['a', 'b', 'c'], []]
```

Comment:
Why would this be the expected output on windows? I thought the idea was to make sure that the same thing is returned?

Reply:
I have written `self.assertNotEqual(expect, result)`. For windows when newline is not set to `''` the OS defaults to `'\r\n'` as mentioned [here](https://docs.python.org/3/library/csv.html#id3).

Reply:
Ah, I didn't cat that it was not equals. That makes sense then.